### ISSUE TITLE
clangd extension cancels the selection of words via mouse

### ISSUE THREAD

**Issue by BOGEDABUDA** [clangd/vscode-clangd#510]
When I works in vscode, I always try to select some words via mouse and then press del to delete this words or use ctrl+D to modify several same sequences in parallel. Meanwhile, I use clangd to help me understand the whole project.
However, after one week ago, when vscode updated itself, the hover information and inlay hints of clangd begin to cancel the selection of words that I select via mouse. It seriously troubles me, but I have no idea about how vscode and its extension work. 
So, I just want to figure out whether this problem comes from the incompatibility of the new vscode and clangd extension?

**Comment by HighCommander4** [comment]
Could you post a screencast that illustrates the issue you're seeing?

**Comment by BOGEDABUDA** [comment]
Thank you for your reply, I take a video to show my problem. I'm sorry to disturb you because I don't  know whether this problem is caused by clangd.

https://github.com/clangd/vscode-clangd/assets/34343583/56de5bc0-8e38-4ce9-a879-1f66553da13a

**Comment by HighCommander4** [comment]
I'm having trouble making out what is happening in that video. Could you please try taking a screencast using something like [Chronicler](https://marketplace.visualstudio.com/items?itemName=arcsine.chronicler)?

**Comment by BOGEDABUDA** [comment]
Of course, note that the selection of word is immediately cancelled at the moment when the hover information pops up.
https://github.com/clangd/vscode-clangd/assets/34343583/fd9998da-036c-4e05-9f20-1803fd173d14

**Comment by HighCommander4** [comment]
A couple of questions:
 * In the scenario depicted in the screnncast, are you selecting words using double-click? And then is the selection cleared without you sending any further input?
 * What vscode version are you using?

**Comment by BOGEDABUDA** [comment]
- I use double-click to select the word and the selection is cleared without any further input.
- I just update the vscode in the morning, but the same problem still exists, the version of my vscode is as follows.
Version: 1.80.2 (user setup)
Commit: 2ccd690cbff1569e4a83d7c43d45101f817401dc
Date: 2023-07-27T20:40:28.909Z
Electron: 22.3.14
ElectronBuildId: 22695494
Chromium: 108.0.5359.215
Node.js: 16.17.1
V8: 10.8.168.25-electron.0
OS: Windows_NT x64 10.0.22631

**Comment by HighCommander4** [comment]
Very strange. The behaviour that I experience is different:

 * Moving my mouse over an identifier is sufficient to activate hover
 * If I click after the hover is shown, the clicking hides the hover
 * If I click before the hover is shown, the hover never appears

Anyways, I think this is very likely to be an issue with the editor, and not related to the clangd server or vscode-clangd plugin.

**Comment by BOGEDABUDA** [comment]
Anyway, thank you for your help. Maybe I should turn to vscode for further solutions.---- Replied Message ----FromNathan ***@***.***>Date07/28/2023 12:54 ***@***.***> ***@***.***>***@***.***>SubjectRe: [clangd/vscode-clangd] clangd extension cancels the selection of words via mouse (Issue #510)
Very strange. The behaviour that I experience is different:

Moving my mouse over an identifier is sufficient to activate hover
If I click after the hover is shown, the clicking hides the hover
If I click before the hover is shown, the hover never appears

Anyways, I think this is very likely to be an issue with the editor, and not related to the clangd server or vscode-clangd plugin.

—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>
[
{
***@***.***": "http://schema.org",
***@***.***": "EmailMessage",
"potentialAction": {
***@***.***": "ViewAction",
"target": "https://github.com/clangd/vscode-clangd/issues/510#issuecomment-1655002751",
"url": "https://github.com/clangd/vscode-clangd/issues/510#issuecomment-1655002751",
"name": "View Issue"
},
"description": "View this Issue on GitHub",
"publisher": {
***@***.***": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
}
]
